draft-ietf-masque-connect-ethernet.md

```diff
@@ -399,8 +399,8 @@ field. Note that this field can be empty.
 Ethernet frames are encoded using HTTP Datagrams with the Context ID set to
 zero. When the Context ID is set to zero, the Payload field contains a full
 Layer 2 Ethernet Frame (from the MAC destination field until the last byte of
-the Frame check sequence field), as defined by IEEE 802.3, with support for
-optional IEEE 802.1Q tagging (see {{vlan-recommendations}}).
+the Frame check sequence field), as defined by IEEE 802.3. A complete
+frame could include include an IEEE 802.1Q tagging (see {{-recommendations}}).
```

Comment:
s/tagging/tag/.

The section being linked to is still called `vlan-recommendations`, so put the `vlan` back.